### PR TITLE
Fix text overflow issue.

### PR DESCRIPTION
**Pull request by KomalBhadani27**
- Refactor code for text display.
- Add view more when text count crosses text-limit (**471** for now).

### CODE REVIEW

File: stamp_generation/generator_input.py [hunk 1]
```diff
@@ -88,8 +88,19 @@ def get_sample_stamp_pages():
...
+                      'treat Covid-19, but the first major study of Covid-19 '
+                      'treatment published in the New England Journal of '
+                      'Medicine (NEJM) found it did not work on severely '
+                      'ill patients in China.\n India has approved the '
+                      'combination of two antiviral drugs used to treat HIV, '
+                      'lopinavir and ritonavir (400 mg and 100 mg, '
+                      'respectively), to treat Covid-19, but the first major '
+                      'study of Covid-19 treatment published in the New '
+                      'England Journal of Medicine (NEJM) found '
+                      'it did not work on severely ill patients in China.',
```
Comment (detel) [inline stamp_generation/generator_input.py:103]: Can you move this to statics?

Comment (KomalBhadani27) [reply stamp_generation/generator_input.py:103]: I did not understand what you exactly mean by moving it to statics. Can you elaborate?

Comment (detel) [reply stamp_generation/generator_input.py:103]: I guess this is for short term. Ignore this comment.

Comment (KomalBhadani27) [reply stamp_generation/generator_input.py:103]: I get it. Yes, this script is just for testing purpose which can be removed once end-to-end integration PR is merged.

**Pull request merged**